hoomd/mpcd/SphereGeometry.h

```diff
@@ -96,10 +94,7 @@ class __attribute__((visibility("default"))) SphereGeometry
              *this condition should never happen 
              */
 
-            if (m_V == 0 && v2 == 0)
-                {
-                throw std::runtime_error("Velocity of shrinking sphere and velocity of particles is zero");
-                }
+            assert(m_V == 0 && v2 == 0);
```

Comment:
This is the opposite of what we want, it asserts that this corner case happens!
```suggestion
            assert(!(m_V == 0 && v2 == 0));
```